### DISCUSSION TITLE
Calendar: Better anniversaries

### DISCUSSION THREAD

**Discussion by GregLeonhardt** [ryanhowdy/fcms#641] [Ideas]
Improvement request:
The only method for recording an anniversary is to add it to the calendar.  There are a few problems here:
1) When entering the calendar event there is no way to enter the year of the anniversary.
2) If you go back and edit in the correct year, when the anniversary comes up the number of years isn't shown in 'Today's Events".
3) The anniversary isn't attached to people and thus not part of the Family Tree.

**Comment by ryanhowdy** [comment]
I think the solution to this would be to:

1. Add a generic add calendar entry link to the top of the calendar that isn't associated with a date, that way you can choose the full date to change.
2. When you add a new anniversary, it will also update a record in the family tree.  But in order to do this, you will have to choose the other person when creating the event.

It might be a good idea to change how the event categories work.  So right now 'Anniversary' is just a category of an event, nothing special really.  But i could make it so that you have to choose the category/type first, and it changes the behavior of the event.  So if you choose anniversary, it will allow you to pick another user that shares the anniversary with you.

**Comment by ryanhowdy** [comment]
Created Issue #642 for this.
